syne_tune/blackbox_repository/conversion_scripts/scripts/hpob_import.py

```diff
@@ -0,0 +1,606 @@
+    "config_space": {
+        "alpha": loguniform(1.0, 2.72),
+        "lambda": loguniform(0.001, 1024.0),
+    },
+}
+SEARCH_SPACE_6767 = {
+    "name": "6767",
+    "config_space": {
+        "eta": uniform(0.001, 1.0),
+        "subsample": uniform(0.1, 1.0),
+        "lambda": uniform(0.001, 1024.0),
+        "alpha": uniform(0.001, 1024.0),
+        "nthread": uniform(0.0, 1.0),
+        "nthread.na": uniform(0.0, 1.0),
+        "nrounds": uniform(0.0, 5000.0),
+        "nrounds.na": uniform(0.0, 1.0),
+        "max_depth": uniform(0.0, 15.0),
+        "max_depth.na": uniform(0.0, 1.0),
+        "min_child_weight": uniform(0.0, 128.0),
+        "min_child_weight.na": uniform(0.0, 1.0),
+        "colsample_bytree": uniform(0.0, 1.0),
+        "colsample_bytree.na": uniform(0.0, 1.0),
+        "colsample_bylevel": uniform(0.0, 1.0),
+        "colsample_bylevel.na": uniform(0.0, 1.0),
+        "booster.ohe.na": 430736,
+        "booster.ohe.gblinear": 620644,
+    },
+}
+SEARCH_SPACE_6794 = {
+    "name": "6794",
+    "config_space": {
+        "num.trees": uniform(0.0, 2000.0),
+        "num.trees.na": uniform(0.0, 1.0),
+        "mtry": uniform(1.0, 1558.0),
+        "sample.fraction": uniform(0.1, 1.0),
+        "min.node.size": uniform(-9.21, 10.72),
+        "min.node.size.na": uniform(0.0, 3195.0),
+        "replace.ohe.FALSE": 623218,
+        "replace.ohe.na": 625345,
+        "respect.unordered.factors.ohe.INVALID": 625034,
+        "respect.unordered.factors.ohe.TRUE": 623529,
+    },
+}
+SEARCH_SPACE_7607 = {
+    "name": "7607",
+    "config_space": {
+        "num.trees": uniform(0.0, 2000.0),
+        "num.trees.na": uniform(0.0, 1.0),
+        "mtry": uniform(1.0, 1775.0),
+        "min.node.size": loguniform(1.0, 44808.0),
+        "sample.fraction": uniform(0.1, 1.0),
+        "replace.ohe.FALSE": 14478,
+        "replace.ohe.na": 14434,
+        "respect.unordered.factors.ohe.INVALID": 14392,
+        "respect.unordered.factors.ohe.TRUE": 14520,
+    },
+}
+SEARCH_SPACE_7609 = {
+    "name": "7609",
+    "config_space": {
+        "num.trees": uniform(0.0, 2000.0),
+        "num.trees.na": uniform(0.0, 1.0),
+        "mtry": uniform(1.0, 1776.0),
+        "min.node.size": loguniform(1.0, 45280.0),
+        "sample.fraction": uniform(0.1, 1.0),
+        "replace.ohe.FALSE": 32943,
+        "replace.ohe.na": 33003,
+        "respect.unordered.factors.ohe.INVALID": 33007,
+        "respect.unordered.factors.ohe.TRUE": 32939,
+    },
+}
+SEARCH_SPACE_5889 = {
+    "name": "5889",
+    "config_space": {
+        "num.trees": loguniform(0.0001, 2000.0),
+        "num.trees.na": uniform(0.0, 1.0),
+        "mtry": loguniform(1.0, 1324.0),
+        "sample.fraction": uniform(0.1, 1.0),
+        "replace.ohe.FALSE": 1215,
+        "replace.ohe.na": 1226,
+    },
+}
+RESOURCE_ATTR = "hp_epoch"
+MAX_RESOURCE_LEVEL = 100
+
+
+def serialize(
+    bb_dict: Dict[str, BlackboxTabular], path: str, metadata: Optional[Dict] = None
+):
```

Comment:
I know why didn't do it for the serialize function for PD1, but could we add a brief comment why we have to overwrite the original serialize function here? Otherwise we might run in the same trap again in the future :-)